src/gengowatcher/orchestration/watcher_job_processor.py

```
@@ -0,0 +1,411 @@
+from .translation_app_queue import (
+    submit_translation_app_task as _submit_translation_app_task,
+)
+from .watcher_config_values import PLACEHOLDER_CONFIG_VALUES
+from .watcher_job_metadata import (
+    derive_lang_pair,
+    derive_word_count,
+)
+
+try:
+    from .translation_app_client import TranslationAppClient
+except Exception:  # pragma: no cover - translation-app optional
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)